client/modules/i18n/startup.js

```diff
@@ -9,6 +9,7 @@ import { Reaction } from "/client/api";
 import { Shops, Translations } from "/lib/collections";
 import * as Schemas from "/lib/collections/schemas";
 import i18next, { packageNamespaces, getLabelsFor, getMessagesFor, i18nextDep } from "./main";
+import {mergeDeep} from "/lib/api";
```

Comment:
```  
  "sort-imports": ["error", {
        "ignoreCase": false,
        "ignoreMemberSort": false,
        "memberSyntaxSortOrder": ["none", "all", "multiple", "single"]
    }],
    "object-curly-spacing": ["error", "always", { "objectsInObjects": true }],
```

However, that's going to change 90 files, as --fix rightfully adds spacing to objects all over the place.  I'm actually cool with that, but it's new... 

Sorts rule isn't quite advanced enough to enforce our rules (I think this basically is just alphabetical).